examples/specs/geo_choropleth.vl.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "width": 500,
+  "height": 300,
+  "mark": "geoshape",
+  "data": {
+    "url": "data/us-10m.json",
+    "format": {
+      "type": "topojson",
+      "feature": "counties"
+    }
+  },
+  "transform": [{
+    "lookup": "id",
+    "from": {
+      "data": {
+        "url": "data/unemployment.tsv",
+        "format": {
+          "type": "tsv"
+        }
+      },
+      "key": "id",
+      "fields": ["rate"]
+    }
+  }],
+  "projection": {
+    "type": "albersUsa"
+  },
+  "encoding": {
+    "color": {
+      "field": "rate",
+      "type": "quantitative"
+    },
+    "opacity": {
```

Comment:
Once you write docs, you might wanna put

"opacity": {"value": 0.7}  

in one line so the spec isn't too long in the docs-- same for projection and other files.